scipy/special/orthogonal.py

```diff
@@ -1371,6 +1371,8 @@ def gegenbauer(n, alpha, monic=False):
     ----------
     n : int
         Degree of the polynomial.
+    alpha : float
+        Parameter, must be greater than -0.5.
```

Comment:
Good catch. Do we know if this parameter has a name in the context of this sort of function?

Reply:
The only context I could find about alpha was the following:

https://www.sciencedirect.com/topics/mathematics/gegenbauer-polynomial

Alpha matters in that depending on its value, you get different kinds of special polynomials.